.github/workflows/main.yml

```diff
@@ -54,13 +54,9 @@ jobs:
           pylint api.user_models
           pylint tests/unit_tests
 
-      - name: Export environment variables
-        run: |
-          echo "SECRET_KEY=$(openssl rand -hex 32)" >> $GITHUB_ENV
-
       - name: Run pytest
         run: |
-          pytest -v tests/unit_tests
+          SECRET_KEY=$(openssl rand -hex 32) SMTP_HOST=smtp.gmail.com SMTP_PORT=465 EMAIL_SENDER=test@kernelci.org EMAIL_PASSWORD=random pytest -vs tests/unit_tests/
```

Comment:
Also, ideally the key should be fixed rather than recreated every time the test is run.  In some particular corner cases, using truly random data in tests might cause false positives that can't be reproduced.

Reply:
See #402